src/Graph.spec.ts

```diff
@@ -19,5 +19,104 @@ describe("boardToGraph", () => {
     expect(GRAPH.nodes["0,1"].isExit).toBe(true);
     expect(GRAPH.edges[0].from).toBe("0,0");
     expect(GRAPH);
+
+  });
+
```

Comment:
What happen if I test "A99" or "Z12" ? is it relevant to add a test to theses cases ?

Reply:
We do not block for the moment the kind of case A99, because our board can vary. And this function should not take this kind of logic in my opinion. 

But if it doesn't match the pattern like 12D, the following test is for that.

Reply:
Ok !